docs/developer_guide.md

```diff
@@ -0,0 +1,42 @@
+# Developer Guide
+
+This repository provides a sandbox environment that can be used to develop and test Continuous Compliance workflows and integrations. 
+
+This document provides information about how to enter the sandbox, configure git and gh authentication, and provides a general overview on the default resources included in the sandbox. 
+
+## Enter the Sandbox Workspace
+
+Run the following commands to access the trestle workspace within the sandbox:
+
+```bash
+make demo-build
+make sandbox-run
+cd trestle-workspace
+```
+
+## Sandbox Auth Helpers
+
+Reference the following guide to configure git and gh authentication within the sandbox: [auth.md](./auth.md)
+
+## Sandbox Utils
+
+The sandbox has several utilities that can be used in Continuous Compliance workflows. These utilities can be found in the `/demo/bin` folder of the sandbox. 
+
+
+## Repository Layout
```

Comment:
This layout is no longer valid. We should probably keep the information about the `automation` work in the `automation` README.md. Maybe just link it here?